examples/nodejs/src/snippets/document/detect-document.ts

```diff
@@ -0,0 +1,24 @@
+import * as ScanbotSDK from "scanbotsdk";
+
+export class DetectDocumentSnippet {
+  public static async run(image: ScanbotSDK.ImageRef): Promise<void> {
+    var scanner_params = new ScanbotSDK.DocumentScannerParameters();
+    scanner_params.acceptedAngleScore = 75;
+    scanner_params.ignoreOrientationMismatch = false;
+
+    var config = new ScanbotSDK.DocumentScannerConfiguration();
```

Comment:
It's also possible to do it like this: `new DocumentScannerConfiguration({parameters: {acceptedAngleScore: 75, ignoreOrientationMismatch: false}})`, same in all the other snippets. I'm not telling that it should be changed everywhere but we should show such a possibility of one line initialization instead of creating all these inner objects manually